Dockerfile

```diff
@@ -86,3 +86,6 @@ RUN LAMBDA_EXECUTOR=local make test
 # clean up temporary files created during test execution
 RUN apk del --purge git;
 RUN rm -rf /tmp/localstack/*elasticsearch* /tmp/localstack.* tests/ /root/.npm/*cache /opt/terraform
+
+# Fix for Centos host OS
+echo "127.0.0.1 localhost.localdomain" >> /etc/hosts
```

Comment:
When the host os is Centos inside docker container `localhost.localdomain` could not be resolved.